src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,936 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+UPDATE schedules SET on_call = false
+  WHERE day = '2023-12-05'
+  AND doctor_id = 1;
+~~~
+
+However, because Session 2 has also acquired a shared lock on these rows, the current transaction is blocked until Session 2 releases its lock.
+
+{{site.data.alerts.callout_success}}
+`FOR SHARE` is not typically used by transactions that intend to update rows, as shown by the following steps. For more information, see [When to use locking reads](#when-to-use-locking-reads).
+{{site.data.alerts.end}}
+  </div>
+
+  <div class="grid-item">
+In Session 2, the previous read confirmed that another doctor is available on `2023-12-05`. Update the schedule to put Betty on leave:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+UPDATE schedules SET on_call = false
+  WHERE day = '2023-12-05'
+  AND doctor_id = 2;
+~~~
+
+However, because Session 1 has also acquired a shared lock on these rows, the current transaction is blocked until Session 1 releases its lock.
+
+Because each concurrent transaction is waiting on the other to release its lock, they will deadlock. Session 2 aborts first with a [`ABORT_REASON_PUSHER_ABORTED`]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#abort_reason_pusher_aborted) error:
+
+~~~
+ERROR: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=85394586 key=/Tenant/2/Table/115/1/19696/1/0 iso=ReadCommitted pri=0.01582601 epo=0 ts=1700690868.143651000,0 min=1700690819.160354000,0 seq=0} lock=true stat=ABORTED rts=1700690868.143651000,0 wto=false gul=1700690819.660354000,0
+~~~
+</div>
+
+  <div class="grid-item">
+With the transaction in Session 2 aborted, Session 1 completes its update to place Abe on leave:
+
+~~~
+UPDATE 1
+~~~
+
+Commit the transaction:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+~~~
+  </div>
+</div>
+
+Read the rows for `2023-12-05` and confirm that Betty is still on call:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM schedules
+  WHERE day = '2023-12-05';
+~~~
+
+~~~
+     day     | doctor_id | on_call
+-------------+-----------+----------
+  2023-12-05 |         1 |    f
+  2023-12-05 |         2 |    t
+~~~
+
+## Known limitations
+
+The following are not yet supported with `READ COMMITTED`:
+
+- Schema changes cannot be performed within explicit `READ COMMITTED` transactions, and will cause transactions to abort. As a workaround, [set the transaction's isolation level](#set-the-current-transaction-to-read-committed) to `SERIALIZABLE`.
+- `READ COMMITTED` isolation cannot be set at the cluster level. As a workaround, run `ALTER ROLE ALL SET default_transaction_isolation = 'read committed'`.
+- `READ COMMITTED` transactions cannot access [`REGIONAL BY ROW`]({% link {{ page.version.version }}/table-localities.md %}#regional-by-row-tables) tables.
```

Comment:
Some RBR tables work, as described in https://github.com/cockroachdb/cockroach/issues/110873. (Do you think it would be too confusing to qualify this?)

Reply:
Rephrased this item to cover that issue, to the best of my understanding.